.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 22.11.0
       - name: Install dependencies
```

Comment:
We normally use only the first two digits to define NodeJS version in GitHub actions, therefore this could be:

```json
node-version: 22
```